.github/workflows/ci-test.yaml

```diff
@@ -34,7 +34,7 @@ jobs:
         ports:
           - 5432:5432
     env:
-      CI_TEST_CONN_STRING: "postgresql://postgres:postgres@127.0.0.1:5432/postgres"
+      CI_TEST_CONN_STRING: "postgresql://postgres:postgres@127.0.0.1:5432/postgres?sslmode=disable"
```

Comment:
What changed so that this switch is needed?

Reply:
The `golang-migrate` (library used for migrations) requires it (it tries to use a ssl connection by default). 

Before, migrations were not run in uni tests. Other parts of the code (e.g. e2e-tests) already have this configured: https://github.com/oasisprotocol/oasis-indexer/blob/4d3a6adc8765e2414fe82fa2e403771e96535118/tests/e2e_regression/e2e_config.yml#L26